latest/src/app/documentation/demos/datagrid/datagrid.demo.html

```diff
@@ -23,5 +23,21 @@ <h4>Omit "select-all" in datagrid header</h4>
         </clr-datagrid>
 
         <clr-code-snippet [clrCode]="noSelectAllExample"></clr-code-snippet>
+
+        <h3>Workaround for Datagrid issues</h3>
+        <p>
+            There are some clarity issues with using the Datagrid.
+            If the datagrid has really few number of rows, then you might face these
+            <a href="https://github.com/vmware/clarity/issues/2683">issues</a>.
+            One workaround for these issues can be to fix the height of datagrid as done in the following code snippet.
+        </p>
+        <clr-code-snippet [clrCode]="fixDatagridHeight"></clr-code-snippet>
+
+        <h3>Usage hints for server driven datagrid</h3>
```

Comment:
The server driven datagrid is already documented here: https://clarity.design/documentation/datagrid/server-driven
If additional information is critical for understanding, please open a pull request on clarity documentation.
Clarity-Addons website should only be used for clarity-addons specific topics